pgmpy/tests/test_causal_discovery/test_PC.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+
+from sklearn.utils.estimator_checks import check_estimator
+
+from pgmpy.causal_discovery import PC
+
+
+class TestPCEstimatorClass(unittest.TestCase):

```

Comment:
We are also trying to move to pytest, so it might be better to update the tests to use pytest instead of unittest.